lstchain/scripts/lstchain_dl1ab.py

```diff
@@ -325,6 +336,12 @@ def main():
                     image_table['image'][ii] = image
                     image_table['peak_time'][ii] = peak_time
 
+                    # use CatB pedestals to estimate the picture threshold
+                    if args.pedestal_cleaning:
+                        threshold_clean_pe = catB_threshold_clean_pe[calib_idx][selected_gain, pixel_index]
+                        threshold_clean_pe[unusable_pixels] = pic_th
```

Comment:
I suggest adding a comment that for unusable pixels the picture threshold is the one set in the config file